.github/workflows/ibis-backends-cloud.yml

```
@@ -58,10 +61,17 @@ jobs:
       - name: download backend data
         run: just download-data
 
+      - uses: google-github-actions/auth@v1
+        if: ${{ matrix.backend.name == 'bigquery' }}
+        with:
+          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+
+      - name: setup snowflake credentials
+        if: ${{ matrix.backend.name == 'snowflake' }}
+        run: echo '${{ secrets.SNOWFLAKE_URL }}' >> "$GITHUB_ENV"
+
```

```suggestion
        run: echo "SNOWFLAKE_URL=${SNOWFLAKE_URL}" >> "$GITHUB_ENV"
        env:
            SNOWFLAKE_URL: ${{ secrets.SNOWFLAKE_URL }}
```
We should use with 'env' option to avoid script injection attacks or injection problems. See: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections

To set the environment variable, you should also pass the environment variable name. https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

---

Fixed, thanks!